README.md

```diff
@@ -117,7 +117,7 @@ export default Ember.Controller.extend({
 ## FullCalendar Scheduler
 
 ### Opting In
-By default, FullCalendar Scheduler is NOT imported. To include it, add the following to your application's `ember-cli-build.js`:
+By default, FullCalendar Scheduler is NOT imported. To include it, add the following to your application's `config/environment.js`:
```

Comment:
I don't believe that this is correct. The example you're showing below in addition to the current code for importing the scheduler is based on settings in `ember-cli-build.js`